src/eventSchemas/models/schemaCodeLangs.ts

```diff
@@ -54,7 +54,7 @@ export function getLanguageDetails(language: SchemaCodeLangs): {
 }
 
 export function supportsEventBridgeTemplates(runtime: Runtime): boolean {
-    return ['python3.7', 'python3.8', 'python3.9', 'go1.x'].includes(runtime)
+    return ['python3.7', 'python3.8', 'python3.9', 'python3.10', 'go1.x'].includes(runtime)
```

Comment:
AFAIR, normally we don't update this. But seems to work: I tried `AWS: Create Lambda SAM Application` with "eventBridge Starter App" and it successfully created a new project. And I see https://github.com/aws/aws-sam-cli-app-templates/tree/master/python3.10/event-bridge , so that further confirms it.